pixi/mocks/mobileFriendlinessCheck/apiResponse.js

```diff
@@ -0,0 +1,59 @@
+const mobileFriendly = {
+  testStatus: {
+    status: 'COMPLETE',
+  },
+  mobileFriendliness: 'MOBILE_FRIENDLY',
+};
+
+const mobileFriendlyWithIssues = {
+  testStatus: {
+    status: 'COMPLETE',
+  },
+  mobileFriendliness: 'MOBILE_FRIENDLY',
+  resourceIssues: [
```

Comment:
Looked into this more. There are two types of issues, `mobileFriendlyIssues` and [`resourceIssues`](https://developers.google.com/webmaster-tools/search-console-api/reference/rest/v1/urlTestingTools.mobileFriendlyTest/run#resourceissue). The former should fail the whole test so does not fall into this `mobileFriendlyWithIssues` edge case, but we should still pipe the findings through to the recommendation so they can improve the mobile friendliness status. For the latter we can note the status and the recommendation should be specific to [blocked resources](https://support.google.com/webmasters/answer/6352293#blocked-resources). /cc @nainar @sebastianbenz as there may be different copy for these.